docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -84,7 +84,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``any``                     | P                               |                                  | ``N`` for non-integer/boolean types                |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``apply``                   | P                               |                                  | ``N`` if ``axis == 0`` or ``func`` is not callable |
+| ``apply``                   | P                               |                                  | ``N`` if ``func`` is not callable                  |
```

Comment:
Can we reference the list of limitations given for `GroupBy.apply` in `groupby_supported.rst`?

Reply:
I looked at the limitations of groupby_supported.rst for GroupBy.apply and they are not hit in the case of Dataframe.apply(axis=0) so I don't think there's anything to document there since this is otherwise an implementation detail.